### PR TITLE
release notes for v1.18.1

### PR DESCRIPTION
**Pull request by jordan40**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -33,32 +33,25 @@ blog:
...
 
-- Validate destinations retrieved from environment variable or destination service.
-- Allow setting the format for the keys of a destination in the `destinations` environment variable as known from the destination service - e. g. `URL` in addition to previously only `url`.
+- Introduce validation for destinations retrieved from environment variable or destination service.
+- Allow configuring the `destinations` environment variable with the same property keys as known from the destination service on SAP Cloud Platform, e.g. now `URL` is considered valid in addition to previously only `url`.
 
 ## fixedIssues
 
-- Fix ordering for complex properties including nested complex properties.
+- Fix ordering for properties, including nested complex properties.
```
Comment (marikaner) [inline CHANGELOG.md]: This changes the meaning. I think what was meant is `Fix ordering for properties of type "complex type", that include properties of type "complex type"`.  Not 100% sure though - @FrankEssenberger ?

File: CHANGELOG.md [hunk 2]
```diff
@@ -33,32 +33,25 @@ blog:
...
 
-release-date:
-docs:
-blog:
-
-## knownIssues
-
--
+release-date: March 26, 2019
+docs: https://sap.github.io/cloud-sdk/
```
Comment (jjtang1985) [inline CHANGELOG.md]:
```suggestion
docs: https://sap.github.io/cloud-sdk/api/
```

**Comment by jordan40** [comment]
> @jordan40 I left a comment. Could you also add a link for the docs of version 1.18.0: https://sap.github.io/cloud-sdk/api/1.18.0

docs for 1.18.0 are included in the HTML version of the release notes:
https://help.sap.com/doc/6c02295dfa8f47cf9c08a19f2e172901/1.0/en-US/js-index.html

**Comment by marikaner** [comment]
> > @jordan40 I left a comment. Could you also add a link for the docs of version 1.18.0: https://sap.github.io/cloud-sdk/api/1.18.0
> 
> docs for 1.18.0 are included in the HTML version of the release notes:
> https://help.sap.com/doc/6c02295dfa8f47cf9c08a19f2e172901/1.0/en-US/js-index.html

@jordan40 Those are the docs for the VDM, I meant the API docs

**Pull request merged**